### PR TITLE
Cash 185 2

### PR DESCRIPTION
**Pull request by stevekinyori**

### CODE REVIEW

File: omod/src/main/webapp/resources/js/screen/bill.js [hunk 1]
```diff
@@ -194,13 +194,20 @@ curl(
...
 				case BillStatus.POSTED:
 				case BillStatus.PAID:
-                    $saveButton.val(__("Adjust Bill"));
-                    $saveButton.click(this.billView.handleAdjustBill);
-                    $printButton.val(__("Print Receipt"));
-                    $printButton.click(function(event) {self.billView.printReceipt(event);
-                        $(this).attr("disabled", "disabled");
-                    });
-                    $printButton.show();
+					//Disable adjust button if this option is enabled.
```
Comment (akmad) [inline omod/src/main/webapp/resources/js/screen/bill.js]: The formatting in this section is still off.

File: omod/src/main/resources/config.xml [hunk 2]
```diff
@@ -69,7 +69,13 @@
 		<property>${project.parent.artifactId}.adjustmentReasonField</property>
 		<description>true/false whether or not the adjustment reason field should be used.</description>
 	</globalProperty>
-	
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.allowBillAdjustments</property>
+		<description>True/false weather or not the adjustment field feature should be turned off.</description>
```
Comment (akmad) [inline omod/src/main/resources/config.xml]: I think the description should be "... should be turn on."

**Comment by akmad** [comment]
Just fix up the remaining text and formatting issues and this looks good!

File: omod/src/main/resources/config.xml (hunk 2 above)
Comment (stevekinyori) [inline omod/src/main/resources/config.xml]: Thats true.. am working on that

File: omod/src/main/webapp/resources/js/screen/bill.js (hunk 1 above)
Comment (stevekinyori) [inline omod/src/main/webapp/resources/js/screen/bill.js]: let me get rid of that comment

Comment (akmad) [inline omod/src/main/webapp/resources/js/screen/bill.js]: The comment is fine... the indentation is messed up for this section of code.

File: omod/src/main/webapp/resources/js/screen/bill.js [hunk 3]
```diff
@@ -194,13 +194,19 @@ curl(
...
 				case BillStatus.POSTED:
 				case BillStatus.PAID:
-                    $saveButton.val(__("Adjust Bill"));
-                    $saveButton.click(this.billView.handleAdjustBill);
-                    $printButton.val(__("Print Receipt"));
-                    $printButton.click(function(event) {self.billView.printReceipt(event);
-                        $(this).attr("disabled", "disabled");
-                    });
-                    $printButton.show();
+					var $allowBillAdjustment = $('#allowBillAdjustment');
```
Comment (akmad) [inline omod/src/main/webapp/resources/js/screen/bill.js]: The comment was not the problem here, please add it back. The issue is still the indentation of the lines that you added. Some of them are ok but some of them are not. Please review them and ensure that they are consistent.

**Comment by akmad** [comment]
Looks good, thanks!

**Pull request merged**